Software-Construction/Factory-And-Abstract-Factory/Graphics/Screen.java

```diff
@@ -0,0 +1,177 @@
+package Graphics;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import Graphics.Shapes.ShapeType;
+
+/**
+ * @author Niraj Gupta
+ * Class to represent the screen on which the shapes are drawn.
+ */
+public class Screen{
+	private List<Shapes> listOfShapes = new ArrayList<>();
+	private final double XMAX, YMAX;

```

Comment:
can XMAX and YMAX be declared as static?

Reply:
No, Because they might be case where we have different screen with different dimensions.